README.md

```diff
@@ -108,6 +108,14 @@ should receive the Rows from in and may send any number of Rows to out. It
 should not return until it has finished all work (received all the Rows it's
 going to receive, sent all the Rows it's going to send).
 
+## Development
+Tests expects the files to be in `$GOPATH/src/gopkg.in/clever/optimus.v3` to work properly
```

Comment:
Also, the README is autogenerated - this should be put in the godocdown template: https://github.com/Clever/optimus/blob/master/.godocdown.template